test/src/blockMonitor/blockCache.test.ts

```diff
@@ -58,413 +58,486 @@ describe("BlockCache", () => {
     beforeEach(async () => {
         db = LevelUp(EncodingDown<string, any>(MemDown(), { valueEncoding: "json" }));
         blockStore = new BlockItemStore<IBlockStub>(db);
-        await blockStore.start();
     });
 
     afterEach(async () => await blockStore.stop());
 
     it("records a block that was just added", async () => {
         const bc = new BlockCache(maxDepth, blockStore);
-        const blocks = generateBlocks(1, 0, "main");
+        await blockStore.withBatch( async () => {
```

Comment:
I dont think we should include batching in these blockcache tests - I think the batching should be tested elsewhere. I think this is a good case for mocking the store, you could also verify that the store was called as expected.

Reply:
Only partially addressed in 2d41006ac2b2374db8e98485e78b5a04508e3358, by removing the code to manage the batch from each test, which is now handled in beforeEach/afterEach blocks.